client/graphql/fragments/userCustomSets.graphql

```diff
@@ -0,0 +1,42 @@
+fragment userCustomSets on User {
```

Comment:
I still prefer a fragment that's reusable in more situations, since this is only usable on the user node. Meaning, I think the fragment would be more reusable if it was only what's inside node {}.